task22/js/main.js

```diff
@@ -68,13 +68,17 @@ const getCreatedTableHeader = (userTableColumn) => {
     const theadElement = document.createElement("thead");
     const tableRowElement = document.createElement("tr");
     const tableHeaderFragment = document.createDocumentFragment();
-    for(const column of Object.values(userTableColumn)){
+    const sortTarget = ["id", "age"];
+    for(const [columnKey, columnValue] of Object.entries(userTableColumn)){
         const tableHeaderElement = createElementWithClassName("th", "table-header");
-        const isSortCategory = column === "ID";
-        const renderSortButton = getCreatedSortButton();
+        const renderSortButtons = getCreatedSortButtons(columnKey);
```

Comment:
nits
I felt that since there was only one place to use it, I didn't need the variable.
```
 if (sortTarget.includes(columnKey)){
        tableHeaderElement.appendChild(getCreatedSortButtons(columnKey));
 }
```

Reply:
It certainly doesn't need to be a variable. Fixed.
[97fc462](https://github.com/seeeen044/JavaScript_handson/pull/24/commits/97fc462242f766c3c6c1e6d490ba51e85d1e3d8a)